proto/storage/compliance_config.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "gogoproto/gogo.proto";
+
+package storage;
+
+message ComplianceConfig {
+    string id   = 1 [(gogoproto.moretags) = 'sql:"pk"'];
```

Comment:
What is the id here? Is it a compliance standard ID, or is it an independent UUID? I am guessing the former. Expanding the var name will drastically improve readability.

Reply:
Yes it is the former. I have  followed the name used for compliance id in other place sin the proto like https://github.com/stackrox/stackrox/pull/5789/files#diff-31845613f38a396813f8a4509ccd6fba1eff28c8439d3defa101d537ddbb5905L32

Reply:
TAL at the following API request object. It is a good example of clear API.

https://github.com/stackrox/stackrox/blob//master/proto/api/v1/compliance_service.proto#L40

https://github.com/stackrox/stackrox/blob/master/proto/api/v1/compliance_service.proto#L48

Now compare it with the following;. Note the wrapping object is named `ComplianceStandardMetadata`, hence, choice of var name `id` seems okay because warping object indicates that it defines metadata for a compliance standard.
https://github.com/stackrox/stackrox/blob/master/proto/api/v1/compliance_service.proto#L28